runtime/rasdump/javadump.cpp

```diff
@@ -1771,6 +1777,43 @@ JavaCoreDumpWriter::writeMemoryCountersSection(void)
 		}
 	}
 
+#ifdef J9ZTPF
+	if (mmap(NULL, 0, 0, MAP_SUPPORTED, 0, 0) != MAP_FAILED)  {
+		rc = mprivatestats(&ztpfNativeStats);
+
+		 if (rc == 0)  {
+			_OutputStream.writeCharacters("========================================");
```

Comment:
This should follow the format of the rest of the javacore file and have tags in the first "column", which is supposed to help with automated parsing of the output.

Lines with spacing, like this one, should use "NULL", the header should use "0SECTION", and then each subsequent level should increment the number and add a unique code, which you can make up based on the content. See any javacore file for how it's done.

Reply:
Okay, I see, didn't realize what the labels meant. Much appreciated.

Updates made.